examples/nested_list/src/list.rs

```diff
@@ -102,8 +102,8 @@ impl Component for List {
         let onmouseover = self.props.on_hover.reform(|_| Hovered::List);
         let onmouseout = self.props.on_hover.reform(|_| Hovered::None);
         html! {
-            <div class="list-container" onmouseout=onmouseout onmouseover=onmouseover>
-                <div class=classes!("list", inactive)>
+            <div class="list-container" onmouseout={onmouseout} onmouseover={onmouseover}>
```

Comment:
After this PR is merged, the next thing I will work on is collapsing this down to:
```suggestion
            <div class="list-container" {onmouseout} {onmouseover}>
```